### DISCUSSION TITLE
Page builder

### DISCUSSION THREAD

**Discussion by Yanngautreau** [cnumr/best-practices-wordpress#17] [Bonnes pratiques] [answered]
L'utilisation d'un page builder (elementor, wpbakery) : vous voulez l'aborder dans service tiers ou dans les thèmes à développer car d'experience c'est une pratique très courante, peut-etre en lien avec les animations (via elementor). Préconisation : opter pour Gutemberg ?

**Comment by aureliegerber** [comment]
Les pages builder que je vois le plus sont elementor et divi. Impossible d'avoir un site avec un eco-index acceptable avec de tels outils. Je préconiserais donc l'utilisation de Gutenberg oui, avec un thème adapté.

**Comment by dNicolle** [comment]
@Yanngautreau , @aureliegerber  (dsl du timing, je n'ai pas vu la surface depuis quelques semaines)
Le souci vient surtout des thèmes qui sont achetés, ils proposent parfoir des thèmes builder par défaut.
Bien sur que la première des recommandations est de faire avec les outils classiques de WP, mais un thème vendu ajoute un tas de fonctionnalités pour palier aux besoins éventuels futurs, et non aux besoin réèls...

@Yanngautreau ta questions est en lien avec https://github.com/cnumr/fiches-wordpress-greenit/blob/main/best-practices-WordPress/fiches/21.%20Comment%20choisir%20son%20th%C3%A8me_.md ?
Ou c'est pour l'ajout d'une fiche dédiée ?

**Comment by florinesueur** [comment]
Je relance cette discussion non cloturée pour pouvoir en faire une issue.
@dNicolle et @Yanngautreau, avez-vous une solution ?

- Ugrader la `BP 21 Choisir son thème` en indiquant aussi dans son titre que l'on parle de page builder + étoffer le paragraphe
- Faire une nouvelle `BP Utiliser le page builder par défaut`

Au vu de la notériété d'Elementor, Divi and co, personnelement, je serais pour une BP supplémentaire.

**Comment by Yanngautreau** [comment, marked as answer]
Je suis pour une fiche dédié, je peux m'en chargé
